apps/src/templates/projects/StartNewProject.jsx

```diff
@@ -0,0 +1,108 @@
+import React, {PropTypes} from 'react';
+import NewProjectButtons from './NewProjectButtons.jsx';
+import i18n from '@cdo/locale';
+import Button from '../Button';
+import color from '../../util/color';
+import {valueOr} from '../../utils';
+
+const styles = {
+  button: {
+    float: 'right',
+    marginRight: 1,
+  },
+  headingStartNew: {
+    paddingRight: 10,
+    paddingBottom: 10,
+    fontSize: 16,
+    fontFamily: '"Gotham 4r"',
+    color: color.charcoal,
+    marginBottom: -10
+  },
+  spacer: {
+    paddingTop: 10,
+    clear: 'both',
+    width: '100%'
+  }
+};
+
+class StartNewProject extends React.Component {
+  static propTypes = {
+    projectTypes: PropTypes.arrayOf(PropTypes.string),
```

Comment:
I'm a little surprised that `projectTypes` isn't required.

Reply:
`projectTypes` determines what we show in the top `NewProjectButtons` "row" in `StartNewProject`, however we fall back to a default (which depends on `canViewAdvancedTools`) if it is not specified. The only place we specify non-default `projectTypes` is in `StageExtras`. I'm open to ways to do this differently or make this clearer.

Reply:
Ah, that makes sense.  I think this is fine, if I'd read a little deeper I would have seen this. :+1: